toolchain/check/eval.cpp

```diff
@@ -227,12 +222,12 @@ static auto GetConstantValue(EvalContext& eval_context, SemIR::InstId inst_id,
   return eval_context.context.constant_values().GetInstId(const_id);
 }
 
-// A type is always constant, but we still need to extract its phase.
+// Gets the type corresponding to the specified type in this evaluation context.
```

Comment:
I'm not sure whether "type" in the above is referring to the input type_id or the returned type_id (I could read it either way for the pair). Maybe:

```suggestion
// Given a type which may refer to a generic parameter, returns the corresponding type in the evaluation context.
```